README.md

```diff
@@ -1,2 +1,16 @@
 # Rueppell
-Rueppell -- is a fox species -- but also is our universal community website. From the ground up.
+Rueppell -- is a fox species living in the Middle-East, Southwestern Asia and, more importantly for us, North Africa -- but it's also our universal community website. From the ground up.
+## How will it be done ?
+### Static Website Generator
+Our use of a Static Website Generator is motivated by our desire to have a CMS-free website and tighten our control of the content as well as our website's security. By having static webpages, we also get rid of real-time data and customized content for users, but that's not really a bad thing since we do not plan to use much of that.
+### And what are Static Website Generators
+Static Website Generator are used to generate static webpages by taking plain files and applying layouts to them. The end result will be a HTML file that will be sent to the website visitor. 
+### Which Static Website Generator engine are we going to use ?
+We're still looking for one ! Jekyll is ruled out for now, but no worries, so many more are available !
+### How to get involved
```

Comment:
I agree, but I wanted to wait until we choose the engine, so we can add instructions for setting up the environment.